menu/iconmenu/IconMenu.c

```diff
@@ -214,20 +214,23 @@ void IconMenu(void) {
 		//If anybody has toggled the xpad left/right, disable the timeout.
 		if (temp!=0) {
 			temp = IoInputDword(0x8008) - COUNT_start;
+			if (temp > (0x369E99*BOOT_TIMEWAIT)) timedOut=1;
```

Comment:
What does this do? What is `temp` here? This could be more self documenting by also moving `IoInputDword(0x8008)` into a function (possibly also apply scaling with `0x369E99` to get rid of magic constants).

Why is this only checked if `temp != 0`?